loops-arrays-objects.js

```diff
@@ -31,10 +32,33 @@ const scores = [
 // A
 // D
 // ==========================================
-
-
-
-
+for(let n = 0; n < scores.length;n++)
+{
+    if(scores[n].score < 60)
+    {
+        scores[n].grade = 'F';
+    }
+    else if(scores[n].score>=60 && scores[n].score < 70)
```

Comment:
Omdat de vorige if false opleverde, hoef je niet nogmaals te checken of de score groter of gelijk is aan 60
```suggestion
    else if(scores[n].score < 70)
```

Reply:
Dit geldt natuurlijk ook voor alle else-if's hieronder